git_pile/git_pile.py

```diff
@@ -1141,7 +1143,14 @@ def cmd_genbranch(args):
         apply_cmd = ["am", "--no-3way"]
         if config.genbranch_committer_date_is_author_date:
             apply_cmd.append("--committer-date-is-author-date")
+
+        env = os.environ.copy()
+        if config.genbranch_user_name:
+            env['GIT_COMMITTER_NAME'] = config.genbranch_user_name
+        if config.genbranch_user_email:
+            env['GIT_COMMITTER_EMAIL'] = config.genbranch_user_email
```

Comment:
My guess about what is happening here is that we are setting committer name and email to "None" which will induce git to use the author name and email, correct, when doing 'git am' etc.? Is there any particular reason why we don't set these to None directly rather than set genbranch_user_name and email to None? Is it just to show the genbranch user name/email is being eliminated?

Reply:
This is an optional feature. If genbranch_user_name and genbranch_user_email are not set in the configuration file, we will not set the environment variables. In this case git will use its normal logic for defining the committer (most likely from whatever is configure as user.email/user.name in the config).

Reply:
OK, since committer info has to be the same in order for git to generate the same SHA's, it seems the only way this can work is if everyone sets the same value for committer name and email in their config files, correct? There is a committer-date-is-author-date but no committer-info-is-author-info.

Anyway, if this is what we have to do that's fine. Please go ahead and merge this when you're ready. Thanks!

Reply:
Yes, there isn't a  `--committer-info-is-author-info` in git am. In future if git has such a flag we can reserve a word like "author" for this setting to start using it.

For sharing the same config among users, just sharing a git config snippet to be included by their config would be enough.